english/forest/pockets/pockets.md

```diff
@@ -0,0 +1,17 @@
+You empty your pants pockets on the grass in front of you. A pocket knife, a
+block of magnesium, a flare, a first aid kit and a machete. Now that's good
+news!
+
+You bandage your leg with supplies from the first aid kit and make a leg
+splint out of two nearby branches and some of the remaining bandages.
+
+It's getting darker. In the distance, you hear something. Could... Could it
+be? Yes! It's a rescue helicopter! If only there was a way to signal them..
+
+You light the flare and wave it over your head! Suddenly you realize, that
+under all this thick foliage, there is no way they are able to see it... Of
+course! You throw the flare at the tree that holds your parachute. Immediately,
+the flare sets it ablaze! You hear the sounds of the helicopter's rotor blades approaching. It lands in a small nearby field. You're saved! As you limp
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.